pages/about.js

```diff
@@ -75,9 +66,29 @@ export default function About(props) {
           </div>
         </div>
       </section>
+      <section className="layout-container">
+        <p className="mb-12 mt-10 xl:w-2/3">
+          {t("getInTouch")}&nbsp;
+          <a
+            className="underline hover:text-canada-footer-hover-font-blue text-canada-footer-font break-words"
+            href={`mailto: ${process.env.NEXT_PUBLIC_NOTIFY_REPORT_A_PROBLEM_EMAIL}`}
+          >
+            {process.env.NEXT_PUBLIC_NOTIFY_REPORT_A_PROBLEM_EMAIL}
+          </a>
+          .
+        </p>
+      </section>
       <CallToAction
         title={t("signupTitle")}
-        description={t("becomeAParticipantDescription")}
+        description={
+          <>
+            {t("becomeAParticipantDescriptionStart")}
+            <Link href="/unsubscribe">
```

Comment:
So because now your passing in raw JSX and HTML we are getting an error with the prop type being of type "object" when it is expecting type "string". This does not mean changing the prop type to an object because then we could pass in literally anything and we don't want that. It should stay a string and work in the way that the TextButtonField works in that you can pass in either HTML (as a string) or any other string.

Reply:
I read about that and it opens vulnerability to cross-site scripting forgery attacks, that's why its called `dangerouslySetInnerHTML`. Also passing in raw HTML disables us from passing JSX an/or components.

So this won't work:
`description="{t(`becomeAParticipantDescriptionStart`)}<Link href='/unsubscribe'><a className='underline'>{t('unsubscribeLinkText')}</a></Link>{t('becomeAParticipantDescriptionEnd')}"`

results in this:
![image](https://user-images.githubusercontent.com/77116011/125837449-6a11b3e2-857e-4187-b6fd-4ee0c54be36b.png)

Reply:
We don't have to worry about cross site scripting because our content is pre-built and pre-rendered on the server before it gets to the client and there is no way for our traffic to be intercepted. The client side will never have access to that, they will only ever see the completed HTML as they are being served a static page. Rogue javascript from other sources has literally no way of getting into that element.

Reply:
Essentially an attacker would have to have access to our server to inject their own JS into it. Which they never will.

Reply:
And to your second point, you write out the HTML as a string, like in the example of the html prop in the TextButtonField storybook. The string would be in the translation files.